capture_gui/plugins/displayplugin.py

```diff
@@ -0,0 +1,148 @@
+import maya.cmds as cmds
+
+from capture_gui.vendor.Qt import QtCore, QtWidgets
+import capture_gui.plugin
+import capture_gui.colorpicker as colorpicker
+
+
+# region GLOBALS
+
+BACKGROUND_DEFAULT = [0.6309999823570251,
+                      0.6309999823570251,
+                      0.6309999823570251]
+
+TOP_DEFAULT = [0.5350000262260437,
+               0.6169999837875366,
+               0.7020000219345093]
+
+BOTTOM_DEFAULT = [0.052000001072883606,
+                  0.052000001072883606,
+                  0.052000001072883606]
+
+COLORS = {"background": BACKGROUND_DEFAULT,
+          "backgroundTop": TOP_DEFAULT,
+          "backgroundBottom": BOTTOM_DEFAULT}
+
+LABELS = {"background": "Background",
+          "backgroundTop": "Top",
+          "backgroundBottom": "Bottom"}
+# endregion GLOBALS
+
+
+class DisplayPlugin(capture_gui.plugin.Plugin):
+
+    """Plugin to apply viewport visibilities and settings"""
+
+    id = "Display Options"
+    label = "Display Options"
+    section = "config"
+    order = 70
+
+    def __init__(self, parent=None):
+        super(DisplayPlugin, self).__init__(parent=parent)
+
+        self._colors = dict()
+
+        self._layout = QtWidgets.QVBoxLayout()
+        self.setLayout(self._layout)
+
+        self.override = QtWidgets.QCheckBox("Override Display Options")
+
+        self.display_type = QtWidgets.QComboBox()
+        self.display_type.addItems(["Solid", "Gradient"])
+
+        # create color columns
+        self._color_layout = QtWidgets.QHBoxLayout()
+        for label, default in COLORS.items():
+            self.add_color_picker(self._color_layout, label, default)
+
+        # populate layout
+        self._layout.addWidget(self.override)
+        self._layout.addWidget(self.display_type)
+        self._layout.addLayout(self._color_layout)
+
+        self.connections()
+
+        # ensure widgets are in the correct enable state
+        self.toggle_override()
+
+    def connections(self):
+        self.override.toggled.connect(self.toggle_override)
```

Comment:
Just wondering, is there a particular reason you often decide to push out the connections of the signals into a separate method? It's never intended to be called additionally, right? Plus the logic in there often is trivial, similar to here.

Are you expecting someone to ever call the `connections` method manually after the widget is instantiated? (Wouldn't that introduce unwanted behavior?)